content/en/Integrations/IntegrationBuilder/How to Guides/map-cobalt-pentest-finding-severity-to-jira-issue-priority.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Map Cobalt Pentest Finding Severity to Jira Issue Priority"
+linkTitle: "Map Cobalt Pentest Finding Severity to Jira Issue Priority"
+weight: 70
+---
+
+{{% pageinfo %}}
+
+This guide is designed to help [**Integration Builder**](/integrations/integrationbuilder/) users map the severity of pentest findings in Cobalt to the priority of new Jira issues.
+Please follow the [Create Jira tickets for Findings guide](/integrations/integrationbuilder/how-to-guides/jira-cloud-migration) before proceeding with this guide.
+
+{{% /pageinfo %}}
+
+## Basic Option
+
+1. Navigate to the __[Cobalt > Jira Cloud] Push pentest finding to Jira__ recipe that you set up while following the previous guide
+
+2. Open the edit view for the recipe by clicking the __Edit__ button
+
+3. Find the __Create Jira Issue__ action in the recipe and click on it
```

Comment:
It looks like we're missing some periods. The following are just a few:
```suggestion
1. Navigate to the __[Cobalt > Jira Cloud] Push pentest finding to Jira__ recipe that you set up while following the previous guide.

2. Open the edit view for the recipe by clicking the __Edit__ button.

3. Find the __Create Jira Issue__ action in the recipe and click on it.
```

Reply:
Updated, thanks!